### PR TITLE
Tests: look at _all_ markdown files for slint snippets

### LINKED ISSUE #2689: Documentation issue: Getting Started page not up to date with 1.0.2
Again a minor one:

- Linux
- slint C++ API
- cmake, FetchContent with GIT_TAG release/1
- rustc 1.69.0 (84c898d65 2023-04-16)
- gcc 11.3.0, with c++17 standard

My file _my_application_ui.slint_ is copy/pasted from slint doc's (Getting Started page)[https://slint-ui.com/releases/1.0.2/docs/cpp/getting_started.html, version 1.0.2.

Generated setter for `my_label` property is marked deleted, which prevents the example code to compile as-is.

```
$ slint-compiler --version
slint-compiler 1.0.2
$ slint-compiler  my_application_ui.slint.slint | grep my_label
    inline auto get_my_label () const = delete /* the property 'my-label' is declared as private. Declare it as 'in', 'out', or 'in-out' to make it public */;
    inline auto set_my_label (const auto &) const = delete /* property 'my-label' is declared as private. Declare it as 'in' or 'in-out' to make it public */;
```

### PR DESCRIPTION
**Pull request by ogoffart**
We were missing some which had errors

Fixes #2689
(Although the error for 2689 was in C++ and would not have been discovered by the test, better to have it parsed anyway)

### CODE REVIEW

File: tests/doctests/build.rs [hunk 1]
```diff
@@ -10,58 +10,76 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
...
-        .chain(std::fs::read_dir(prefix.join("docs/language/src/recipes"))?)
-        .chain(std::fs::read_dir(prefix.join("docs"))?)
+    for entry in walkdir::WalkDir::new(&prefix)
+        .follow_links(false)
+        .into_iter()
+        .filter_entry(|entry| entry.file_name() != "target")
     {
         let entry = entry?;
         let path = entry.path();
         if path.extension().map_or(true, |e| e != "md") {
```
Comment (tronical) [inline tests/doctests/build.rs:20]: Just a drive-by thought: perhaps this can be folded into filter_entry?

**Review by tronical — APPROVED** [review]

File: tests/doctests/build.rs (hunk 1 above)
Comment (ogoffart) [reply tests/doctests/build.rs:20]: filter_entry also make sure we do not recurse into it.
